diff --git a/include/lib/KD2/SMTP.php b/include/lib/KD2/SMTP.php
index fec5b92..723a3fc 100644
--- a/include/lib/KD2/SMTP.php
+++ b/include/lib/KD2/SMTP.php
@@ -187,7 +187,7 @@ class SMTP
 
 	public function connect(): void
 	{
-		$this->log = '';
+		/* $this->log = ''; */
 		$this->conn = stream_socket_client($this->server . ':' . $this->port, $errno, $errstr, $this->timeout);
 		stream_set_timeout($this->conn, $this->timeout); 
